drag_and_drop_v2/public/js/drag_and_drop.js

```diff
@@ -2002,7 +2083,9 @@ function DragAndDropBlock(runtime, element, configuration) {
             problem_html: configuration.problem_text,
             show_problem_header: configuration.show_problem_header,
             show_submit_answer: configuration.mode == DragAndDropBlock.ASSESSMENT_MODE,
-            show_show_answer: configuration.mode == false,
+            show_show_answer: (configuration.mode == DragAndDropBlock.ASSESSMENT_MODE && 
+                configuration.item_sizing == DragAndDropBlock.FREE_SIZING),
+            show_feedback_bar: !canShowFeedbackBar(),
```

Comment:
Please update `canShowFeedbackBar` to return the correct result so it doesn't need to be negated.